### PR TITLE
blackmagic-desktop-video,decklink: 14.0.1a2 -> 14.1a1

### PR DESCRIPTION
**Pull request by WheelsForReals**
## Description of changes

Updating to the latest version.

Desktop Video 14.1 release notes (2024-07-22): https://www.blackmagicdesign.com/support/readme/93b33ad64a244cd5b95ec9d373e8d2e1

I did some basic testing in OBS with my Intensity Pro 4K and everything seems to be working without any issues.

(Apologies if I missed anything as far as contributing etiquette goes. I'm still pretty new to NixOS and this is my first PR.)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by SigmaSquadron — APPROVED** [review]
I see no issues with this. Good work!

Are you sure you don't want to maintain this package in the future though?

**Comment by WheelsForReals** [comment]
I feel vastly underqualified to be added as a maintainer for this, to be completely honest. I have very basic knowledge of C and know little to nothing about kernel modules, drivers, the build system, etc.

I think all I'd really be good for is making trivial version updates like this.

**Comment by SigmaSquadron** [comment]
That's quite alright! The derivations/learning materials will still be here if you change your mind.

All that's left now is to wait for Naxdy's review.

---

Result of `nixpkgs-review pr 332599` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>24 packages built:</summary>
  <ul>
    <li>blackmagic-desktop-video</li>
    <li>linuxKernel.packages.linux_4_19.decklink</li>
    <li>linuxKernel.packages.linux_4_19_hardened.decklink</li>
    <li>linuxKernel.packages.linux_5_10.decklink</li>
    <li>linuxKernel.packages.linux_5_10_hardened.decklink</li>
    <li>linuxKernel.packages.linux_5_15.decklink</li>
    <li>linuxKernel.packages.linux_5_15_hardened.decklink</li>
    <li>linuxKernel.packages.linux_5_4.decklink</li>
    <li>linuxKernel.packages.linux_5_4_hardened.decklink</li>
    <li>linuxKernel.packages.linux_6_1.decklink</li>
    <li>linuxKernel.packages.linux_6_10.decklink</li>
    <li>linuxKernel.packages.linux_6_1_hardened.decklink</li>
    <li>linuxKernel.packages.linux_6_6.decklink</li>
    <li>linuxKernel.packages.linux_hardened.decklink (linuxKernel.packages.linux_6_6_hardened.decklink)</li>
    <li>linuxKernel.packages.linux_6_8.decklink</li>
    <li>linuxKernel.packages.linux_6_8_hardened.decklink</li>
    <li>linuxKernel.packages.linux_6_9.decklink</li>
    <li>linuxKernel.packages.linux_6_9_hardened.decklink</li>
    <li>linuxKernel.packages.linux_latest_libre.decklink</li>
    <li>linuxKernel.packages.linux_libre.decklink</li>
    <li>linuxKernel.packages.linux_lqx.decklink</li>
    <li>linuxKernel.packages.linux_xanmod.decklink</li>
    <li>linuxKernel.packages.linux_xanmod_latest.decklink (linuxKernel.packages.linux_xanmod_stable.decklink)</li>
    <li>linuxKernel.packages.linux_zen.decklink</li>
  </ul>
</details>

**Comment by Naxdy** [comment]
Changes look good, gonna test it on my hardware later and approve if it doesn't explode :)

**Review by Naxdy — APPROVED** [review]
All good on my end!

**Pull request merged**
